src/components/dropdown/nz-dropdown.component.ts

```diff
@@ -7,10 +7,17 @@ import {
   Renderer2,
   ContentChild,
   Output,
-  EventEmitter, AfterViewInit
+  EventEmitter,
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef
 } from '@angular/core';
-import { Subject } from 'rxjs/Subject';
-import { debounceTime } from 'rxjs/operator/debounceTime';
+import 'rxjs/add/operator/debounceTime';
+import 'rxjs/add/operator/do';
+import 'rxjs/add/operator/merge';
```

Comment:
Library should not use rxjs patching mechanism as suggested in https://github.com/angular/material2/pull/5314